### PR TITLE
app-misc/scrub: EAPI=6 bump, update description

### PR DESCRIPTION
**Pull request by jonasstein**
fixes https://bugs.gentoo.org/show_bug.cgi?id=533226

### CODE REVIEW

File: app-misc/scrub/scrub-2.6.1-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="6"
```
Comment (SoapGentoo) [inline app-misc/scrub/scrub-2.6.1-r1.ebuild]: this one is already `EAPI=6`, any reason for the revbump?

**Comment by monsieurp** [comment]
what in the world is blocking this PR from landing in the tree?

**Comment by jonasstein** [comment]
Hi, that was +4 months ago. I can not remember and do not have the time to look at it now. You can change it to your needs if you want. Best, JS

**Comment by mgorny** [comment]
I see no difference from the files in ::gentoo, so I suppose it's all been applied, one way or another.

**Pull request closed without merging**